packages/expo/android/src/main/java/expo/modules/ExpoReactHostFactory.kt

```diff
@@ -117,4 +128,64 @@ object ExpoReactHostFactory {
     }
     return reactHost as ReactHost
   }
+
+  @OptIn(UnstableReactNativeAPI::class)
+  @JvmStatic
+  fun getDefaultReactHost(
+    context: Context,
+    packageList: List<ReactPackage>,
+    jsMainModulePath: String = ".expo/.virtual-metro-entry",
+    jsBundleAssetPath: String = "index.android.bundle",
+    jsBundleFilePath: String? = null,
+    jsRuntimeFactory: JSRuntimeFactory? = null,
+    useDevSupport: Boolean = ReactBuildConfig.DEBUG,
+    bindingsInstaller: BindingsInstaller? = null,
+  ): ReactHost {
```

Comment:
Basically the same function as `createFromReactNativeHost()` but without relying on ReactNativeHost